apps.yml

```
@@ -1,1035 +1,1886 @@
+    url: https://auth.mozilla.auth0.com/samlp/Hy16dlY3Kmw373jWKVeMGulVUTt1y1KX
+    logo: servicenow.png
+    authorized_users: []
+    authorized_groups:
+    - team_mofo
+    - hris_is_staff
+    - team_moco
+    display: false
+- application:
+    name: ServiceNow - Stage
+    client_id: DIx9YBxmtAWGGSV6nc21wSEXuymRSvY9
+    op: auth0
+    url: https://auth.mozilla.auth0.com/samlp/DIx9YBxmtAWGGSV6nc21wSEXuymRSvY9
+    logo: servicenow.png
+    authorized_users: []
+    authorized_groups:
+    - team_mofo
+    - hris_is_staff
+    - team_moco
+    display: false
+- application:
+    name: ServiceNow - Test
+    client_id: dPlqnxe8phbjbhm80W1Ry8SzSGS3ijuh
+    op: auth0
+    url: https://auth.mozilla.auth0.com/samlp/dPlqnxe8phbjbhm80W1Ry8SzSGS3ijuh
+    logo: servicenow.png
+    authorized_users: []
+    authorized_groups:
+    - team_mofo
+    - hris_is_staff
+    - team_moco
+    display: false
+- application:
+    name: ServiceNow - Upgrade
+    client_id: axoW08Nny1HXe0qcP0416YGrmYtfgdTQ
+    op: auth0
+    url: https://auth.mozilla.auth0.com/samlp/axoW08Nny1HXe0qcP0416YGrmYtfgdTQ
+    logo: servicenow.png
+    authorized_users:
+    - moc+servicenow@mozilla.com
+    - moc-sso-monitoring@mozilla.com
+    authorized_groups:
+    - team_mofo
+    - moc_service_accounts
+    - hris_is_staff
+    - team_moco
+    display: false
+- application:
+    name: Smartsheet
+    client_id: l094SuNKnntJaVwkUU2tWxOsEtx5h3Oo
+    op: auth0
+    url: https://app.smartsheet.com/b/orgsso/C5AE787951B226D893639096D1863484
+    logo: smartsheet.png
+    authorized_users: []
+    authorized_groups:
+    - team_mofo
+    - hris_is_staff
+    - team_moco
+    display: true
+    vanity_url:
+    - /smartsheet
+    - /smartsheets
+- application:
+    name: Snippets
+    client_id: A7GAcuN9gE9x3H186dKQgzS3jsV9Qmgp
+    op: auth0
+    url: https://snippets-admin.us-west.moz.works/oidc/authenticate/
+    logo: snippets.png
+    authorized_users: []
+    authorized_groups:
+    - team_mofo
+    - service_snippets
+    - hris_is_staff
+    - team_moco
+    display: true
+- application:
+    name: Splunk
+    client_id: EUmKs3owmNdeDWxZ4CJIeSGM5ez3Suav
+    op: auth0
+    url: https://auth.mozilla.auth0.com/samlp/EUmKs3owmNdeDWxZ4CJIeSGM5ez3Suav
+    logo: splunk.png
+    authorized_users: []
+    authorized_groups:
+    - splunk_admin
+    - hris_is_staff
+    - netops
+    - hris_dept_it
+    - team_mofo
+    - team_moco
+    display: true
+    vanity_url:
+    - /splunk
+- application:
+    name: StatusPage
+    client_id: dc50KcxGnMPBOSlE5QLYaDRmfrO7oXhq
+    op: auth0
+    url: https://auth.mozilla.auth0.com/samlp/dc50KcxGnMPBOSlE5QLYaDRmfrO7oXhq
+    logo: statuspage.png
+    authorized_users: []
+    authorized_groups:
+    - team_mofo
+    - hris_is_staff
+    - team_moco
+    display: true
+    vanity_url:
+    - /statuspage
+- application:
+    name: support.allizom.org:admin
+    client_id: Wz5oO6y8oJ35Yq1B91aC4pkwlXdes7jR
+    op: auth0
+    url: https:
```

```suggestion
    AAL: MEDIUM
```

At @akatsoulas  request